src/ripple/nodestore/impl/DatabaseShardImp.h

```diff
@@ -229,8 +253,14 @@ class DatabaseShardImp : public DatabaseShard
     // pending shards will be stored on the filesystem. A value of
     // boost::none indicates that the corresponding shard is not held
     // by the database.
-    boost::optional<std::uint32_t> latestShardIndex_;
-    boost::optional<std::uint32_t> secondLatestShardIndex_;
+    std::optional<std::uint32_t> latestShardIndex_;
+    std::optional<std::uint32_t> secondLatestShardIndex_;
+
+    // Struct used to record database import progress
```

Comment:
Consider renaming to "Struct used for node store import progress"

Reply:
Done